src/WordSelector/WordSelector.test.js

```diff
@@ -4,10 +4,19 @@ import { screen, render, waitFor } from '@testing-library/react';
 
 describe('WordSelector', () => {
 
-  it('should render the default layout', () => {
+  describe('Testing the Heading', () => {
+    it('should render the default heading if the `isGenerator` prop is true', () => {
 
-    render(<WordSelector />);
+      render(<WordSelector isGenerator={true}/>);
+  
+      expect(screen.getByText('Type a word for your opponent to guess!')).toBeInTheDocument();
+    })
 
-    expect(screen.getByText('Type a word for your opponent to guess!')).toBeInTheDocument();
+    it('should not render the default heading if the `isGenerator` prop is true', () => {
+
+      render(<WordSelector />);
```

Comment:
Also, I know it renders the same thing, but I think for dev empathy we should still pass the prop of "isGenerator={false}" into here.